packages/calling-stateful-client/src/StatefulCallClient.ts

```diff
@@ -193,7 +193,9 @@ class ProxyCallClient implements ProxyHandler<CallClient> {
             const callAgent = await target.createCallAgent(...args);
             this._callAgent = callAgentDeclaratify(callAgent, this._context, this._internalContext);
```

Comment:
is it possible for teamsCallAgent to also take in the displayname? we take in a displayname in two different ways based on if its teams or acs?

Reply:
I suppose its ok if we are only going into public preview as we could change this in the future. Its just odd that we have two different ways of setting display name if your using acs or teams